.circleci/config.yml

```diff
@@ -258,6 +258,46 @@ jobs:
               run --rm --no-deps app \
                 black src/richie/apps src/richie/plugins sandbox --check
 
+  extract-back-strings:
+    machine: true
+    working_directory: ~/fun
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - docker-debian-images-dev-{{ .Revision }}
+      - run:
+          name: Load images to docker engine
+          command: |
+            docker load < docker/images/dev/richie.tar
+      - run:
+          name: Generate a POT file from strings extracted from the project
+          command: make messages
+      - persist_to_workspace:
+          root: ~/fun/src/richie/locale
+          paths:
+            - django.pot
+
+  upload-i18n-strings:
+    docker:
+      - image: circleci/node:9-browsers
+    working_directory: ~/richie
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+          - v3-front-dependencies-{{ checksum "package.json" }}
+      - attach_workspace:
+          at: ~/fun/src/richie/locale
+      - run:
+          name: Use our node script to upload i18n backend strings file
+          command: node i18n/scripts/upload.js 'richie-back.pot' '/../../../fun/src/richie/locale/django.pot'
+      - attach_workspace:
+          at: ~/richie/src/richie-front/i18n
```

Comment:
We need more consistency in our paths (_i.e._ `~/richie`, `~/fun`), but this is not the purpose of this work...